dashboard/streamlit_graphs/sales_over_time.py

```diff
@@ -60,19 +73,24 @@ def fetch_sales_within_date_range(connection, start_date, end_date) -> pd.DataFr
 
 
 def plot_sales_per_hour(connection: extensions.connection,
-                        start_date: datetime, end_date: datetime) -> alt.Chart | None:
+                        start_date: datetime, end_date=None) -> alt.Chart | None:
     """
     Plots a line graph of sales per hour for the current day.
     """
     try:
         sales_data = fetch_sales_within_date_range(
             connection, start_date, end_date)
         if sales_data.empty:
-            st.error("No data available to display")
+            st.error("EMPTY - No data available to display")
```

Comment:
Is 'EMPTY' a sufficient error explanation?

Reply:
see above comment